site2/website-next/release-notes/versioned/pulsar-cs-2.4.1.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Why do we need to make changes on these files (under `release-notes/versioned`.

Reply:
These are newly added, not modified. They are added because they are missing.

Reply:
Is their missing related to this patch? I don't know why they're missed in the first place and why we couple these changes in the patch to resolve the location of reference docs.

Reply:
No, I also have no idea why these docs are missing. I added these to avoid 404 links in the Release Notes page.

Reply:
So where do you add these files from?